src/sentry/workflow_engine/processors/detector.py

```diff
@@ -58,11 +66,34 @@ def process_detectors(
                         "group_key": result.group_key,
                     },
                 )
+                continue
+
+            if result.result is not None:
+                occurrence, status_change = None, None
+                if isinstance(result.result, IssueOccurrence):
+                    occurrence = (
+                        result.result if isinstance(result.result, IssueOccurrence) else None
```

Comment:
Do you need to check `isinstance(result.result, IssueOccurrence)` twice?

Reply:
Oops, left this in place from before I switched this over to an if/else